WTWSMS/history/titles/k_nicene.txt

```diff
@@ -0,0 +1,14 @@
+37.1.1={
+	
+	liege="e_byzantium"
+	
+
+	active = yes
+
+	
+
+	government = theocracy_government
+	
+	succession = succ_open_elective
+
+}
```

Comment:
That is perfectly intended though! Nicene had five patriarches, not one head in that sense.